src/const.h

```diff
@@ -519,8 +519,20 @@ enum PlayerFlags : uint64_t {
 
 #define CHANNEL_GUILD 0x00
 #define CHANNEL_PARTY 0x01
+
+#define CHANNEL_TUTOR 0x02
```

Comment:
Remove all the defines from this to CHANNEL_GAMEMASTER, their ids are customizable in chatchannels.xml and should not be hardcoded. CHANNEL_CAST should be 0xFFFE.